src/Client/Web/Bit.Client.Web.BlazorUI.Tests/Breadcrumb/BitBreadcrumbTest.razor

```diff
@@ -0,0 +1,25 @@
+﻿<CascadingValue Value="Visual">
+    <BitBreadcrumb DividerAs="DividerAs"
+                   Style ="Style"
+                   Class="Class"
+                   Visibility="Visibility"
+                   Items="Items"
+                   MaxDisplayedItems="MaxDisplayedItems"
+                   OnRenderOverflowIcon="OnRenderOverflowIcon"
+                   OverflowAriaLabel="OverflowAriaLabel"
```

Comment:
Yes, of course, I have tested them and there is no problem.

![class1](https://user-images.githubusercontent.com/46419023/161525035-aae6ea92-c26a-44ea-836e-ddb5d5b5174c.JPG)

Reply:
in this image, you created an instance of the BitBreadCrumb component directly and not the test component (BitBreadCrumbTest), and directly assigned the parameter with the correct value.
If you try and use the test component you will see that the string value you provided will be used instead of the actual variables.